src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -104,6 +105,8 @@ def run_mdtest(self, manager, processes, display_space=True, pool=None, out_queu
             object: result of job manager run
         """
         env = self.mdtest_cmd.get_default_env(str(manager), self.client_log)
+        if self.intercept_lib:
+            env['LD_PRELOAD'] = '/usr/lib64/libpil4dfs.so'
```

Comment:
We're trying to move away from the practice of setting magic `self.*` variables to manipulate function behavior, because it's difficult to maintain/refactor/track/etc. It's better for the test to do `self.mdtest_cmd.env['LD_PRELOAD'] = '/usr/lib64/libpil4dfs.so'` before running mdtest.

Reply:
Thank you very much! Fixed as you suggested.